### ISSUE TITLE
Exception when starting Intellij IDEA

### ISSUE THREAD

**Issue by knoxfighter** [DemonWav/MinecraftDevIntelliJ#671]
Please include the following information in all bug reports:

* Minecraft Development for IntelliJ plugin version
2019.3-1.3.2

* IntelliJ version
Intellij IDEA 2019.3.1 (Ultimate Edition)

* Operating System (this has come up enough times to be important)
Ubuntu 18.04

This error occurres every time, i start the intellij IDEA 2019.3.1. This error wasn't there in Intellij IDEA 2019.2

Stacktrace:
```
com.intellij.diagnostic.PluginException: File type detector com.demonwav.mcdev.nbt.filetype.NbtFileTypeDetector@7c923782 does not implement getDetectedFileTypes(), leading to suboptimal performance. Please implement the method. [Plugin: com.demonwav.minecraft-dev]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:405)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:58)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.getDetectorsForType(FileTypeManagerImpl.java:1129)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.isFileOfType(FileTypeManagerImpl.java:1183)
	at com.intellij.lang.aspectj.psi.presentation.AjFileIconPatcher.patchIcon(AjFileIconPatcher.java:21)
	at com.intellij.util.IconUtil.lambda$static$0(IconUtil.java:154)
	at com.intellij.ui.IconDeferrerImpl.deferImpl(IconDeferrerImpl.java:83)
	at com.intellij.ui.IconDeferrerImpl.defer(IconDeferrerImpl.java:73)
	at com.intellij.util.IconUtil.getIcon(IconUtil.java:182)
	at com.intellij.psi.impl.ElementBase.getElementIcon(ElementBase.java:220)
	at com.intellij.psi.impl.ElementBase.doComputeIconNow(ElementBase.java:110)
	at com.intellij.psi.impl.ElementBase.lambda$computeIconNow$2(ElementBase.java:97)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:125)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:114)
	at com.intellij.psi.impl.ElementBase.computeIconNow(ElementBase.java:97)
	at com.intellij.psi.impl.ElementBase.lambda$static$0(ElementBase.java:41)
	at com.intellij.ui.CoreIconManager.lambda$createDeferredIcon$0(CoreIconManager.java:38)
	at com.intellij.ui.DeferredIconImpl.evaluate(DeferredIconImpl.java:264)
	at com.intellij.ui.DeferredIconImpl.lambda$null$0(DeferredIconImpl.java:152)
	at com.intellij.ui.IconDeferrerImpl.evaluateDeferred(IconDeferrerImpl.java:111)
	at com.intellij.ui.DeferredIconImpl.lambda$null$1(DeferredIconImpl.java:152)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:997)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:73)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:127)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:116)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:73)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:94)
	at com.intellij.ui.DeferredIconImpl.lambda$paintIcon$4(DeferredIconImpl.java:151)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:190)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by RedNesto** [comment]
This is not an issue since 2020.1: `getDetectedFileTypes()` is [deprecated](https://github.com/JetBrains/intellij-community/blob/f51ed1627777b0fefbff4af9fd66a1d87420701d/platform/core-api/src/com/intellij/openapi/fileTypes/FileTypeRegistry.java#L159) and the warning [removed](https://github.com/JetBrains/intellij-community/commit/b2edd47e1edad389fc60741945941ef47c783175#diff-a2590eb4630da07c27f3010a7688c35b).

I don't think it's worth implementing it for 2019.3 since there are not visible issues with it not being implemented and the first 2020.1 EAP has been released last week (well, at least I won't be the one doing it.).

**Comment by DenWav** [comment]
I went ahead and fixed it because it was so simple. Fixed in `1.3.3`.
